products-app/products-fe
# PR: add app config

# Description

Add tests config and fix transpiler.

# Highlights

- Add vitest package for tests;
- Add testing library for frontend test;
- Add package to check the test coverage;
- Configure the app for tests;
- Change package @vitejs/plugin-react to @vitejs/plugin-react-swc;
- Update README.
